src/org/zaproxy/zap/extension/params/ExtensionParams.java

```diff
@@ -350,7 +351,15 @@ private void persist(HtmlParameterStats param) {
 						param.getId(), param.getTimesUsed(), setToString(param.getFlags()), setToString(param.getValues()));
 			}
 		} catch (DatabaseException e) {
-			logger.error(e.getMessage(), e);
+			if (e.getCause() instanceof SQLDataException) {

```

Comment:
Should I also (or only) check that `e.getCause().getMessage().contains("truncation");`?

Reply:
Probably better.